src/components/studies/StudyBuilder.tsx

```diff
@@ -94,19 +94,18 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
   ...otherProps
 }) => {
   const classes = useStyles()
-  let { id, section: _section } =
-    useParams<{
-      id: string
-      section: StudySection
-    }>()
+  let { id, section: _section } = useParams<{
+    id: string
+    section: StudySection
+  }>()
   const [section, setSection] = React.useState(_section)
   const [error, setError] = React.useState<string[]>([])
+  const [schedulerError, setSchedulerError] = React.useState<string[]>([])
```

Comment:
do you need a separate schedulerError? Can you just use error? When the scheduler is displayed all the errors will be the scheduler error. You can change the code around the ln 383 and check the section and don't display the errors the the studybuilder if the the section is scheduler